tests/h/search/index_test.py

```diff
@@ -94,6 +102,111 @@ def test_you_can_filter_annotations_by_creation_time(self, factories, index, sea
         response = search.filter("range", created={"gte": before}).execute()
         assert SearchResponseWithIDs([annotation.id]) == response
 
+    def test_you_can_filter_annotations_by_updated_time(self, factories, index, search):
+        update_time = datetime.datetime.now()
+        annotation = factories.Annotation.build(id="test_annotation_id", updated=update_time)
+
+        index(annotation)
+
+        response = search.filter("range", updated={"gte": update_time}).execute()
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_id(self, factories, index, search):
+        annotation = factories.Annotation.build(id="test_ann_id")
+
+        index(annotation)
+
+        response = search.filter("term", id="test_ann_id").execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_user(self, factories, index, search):
+        annotation = factories.Annotation.build(userid="acct:someone@example.com")
+
+        index(annotation)
+
+        response = search.filter("term", user="acct:someone@example.com").execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
+
+    def test_you_can_filter_annotations_by_user_raw(self, factories, index, search):
+        annotation = factories.Annotation.build(userid="acct:someone@example.com")
+
+        index(annotation)
+
+        response = search.filter("term", user_raw="acct:someone@example.com").execute()
+
+        assert SearchResponseWithIDs([annotation.id]) == response
```

Comment:
What's the difference between `"user"` and `"user_raw"`? It looks like `"user"` uses the user analyzer whereas `"user_raw"` doesn't:

https://github.com/hypothesis/h/blob/61fe02289488e0bf88d54aa37dae51a8300c00a5/h/search/config.py#L47-L4

This must enable some kind of search filtering on `"user"`

Reply:
It looks like we are using the analysed `user`field for searching and the `user_raw` field in UsersAggregation in query.py